login-workflow/example/src/screens/AppRouter.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { useState } from 'react';
+import React from 'react';
 import {
     AccountDetailsScreen,
     AuthContextProvider,
     CreateAccountScreen,
     CreatePasswordScreen,
     ContactSupportScreen,
     EulaScreen,
-    ExperimentalAuthGuard,
+    ReactAuthGuard,
```

Comment:
These should be ReactRouterAuthGuard and ReactRouterGuestGuard (they are for use with react-router only)